extensions/qute/runtime/src/main/java/io/quarkus/qute/runtime/VariantTemplateProducer.java

```diff
@@ -166,8 +172,8 @@ public TemplateVariants(Map<Variant, String> variants, String defaultTemplate) {
     }
 
     static String parseMediaType(String suffix) {
-        // TODO support more media types...
-        if (suffix.equalsIgnoreCase(".html")) {
+        // TODO we need a proper way to parse the media type
+        if (suffix.equalsIgnoreCase(".html") || suffix.equalsIgnoreCase(".htm")) {
             return "text/html";
         } else if (suffix.equalsIgnoreCase(".xml")) {
             return "text/xml";
```

Comment:
Shouldn't we apply that one for XML too?

Reply:
Yes, we could.